puppet/modules/slave/manifests/init.pp

```diff
@@ -150,13 +150,13 @@
     source => 'puppet:///modules/slave/JSON.sh',
   }
 
-  # nodejs/npm for Katello JavaScript tests
-  # packages only really available on EL6+
+  # nodejs/npm for JavaScript tests
   if $::osfamily == 'RedHat' {
-    package { 'npm':
-      ensure  => present,
-      require => Package['bzip2'],
-    } -> Package <| provider == 'npm' |>
+    class { 'nodejs':
+      repo_url_suffix       => '12.x',
+      nodejs_package_ensure => latest,
+      npm_package_ensure    => latest,
+    }
```

Comment:
`-> Package <| provider == 'npm' |>` is IMHO still needed, or does the nodejs class handle this on its own?

Reply:
Updated